storage/txcache/txCache.go

```diff
@@ -101,12 +102,21 @@ func (cache *TxCache) GetByTxHash(txHash []byte) (*WrappedTransaction, bool) {
 // It returns at most "numRequested" transactions
 // Each sender gets the chance to give at least "batchSizePerSender" transactions, unless "numRequested" limit is reached before iterating over all senders
 func (cache *TxCache) SelectTransactions(numRequested int, batchSizePerSender int) []*WrappedTransaction {
-	result := cache.doSelectTransactions(numRequested, batchSizePerSender)
+	result := cache.doSelectTransactions(numRequested, batchSizePerSender, math.MaxUint64)
 	go cache.doAfterSelection()
 	return result
 }
 
-func (cache *TxCache) doSelectTransactions(numRequested int, batchSizePerSender int) []*WrappedTransaction {
+// SelectTransactionsWithBandwidth selects a reasonably fair list of transactions to be included in the next miniblock
+// It returns at most "numRequested" transactions
+// Each sender gets the chance to give at least bandwidthPerSender gas worth of transactions, unless "numRequested" limit is reached before iterating over all senders
+func (cache *TxCache) SelectTransactionsWithBandwidth(numRequested int, batchSizePerSender int, bandwidthPerSender uint64) []*WrappedTransaction {
```

Comment:
why do we need a separate endpoint ? the old one could be deleted.

Reply:
right, no longer used, except in some tests, removed.